x-pack/plugins/canvas/public/expression_types/arg_types/series_style/index.js

```diff
@@ -4,17 +4,25 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import React from 'react';
 import PropTypes from 'prop-types';
 import { lifecycle } from 'recompose';
 import { get } from 'lodash';
+import { FormattedMessage, injectI18n } from '@kbn/i18n/react';
 import { templateFromReactComponent } from '../../../lib/template_from_react_component';
 import { SimpleTemplate } from './simple_template';
 import { ExtendedTemplate } from './extended_template';
 
 const EnhancedExtendedTemplate = lifecycle({
   formatLabel(label) {
     if (typeof label !== 'string') this.props.renderError();
-    return `Style: ${label}`;
+    return (
+      <FormattedMessage
+        id="xpack.canvas.expressionTypes.seriesStyle.styleLabelTitle"
```

Comment:
xpack.canvas.expressionTypes.seriesStyle => xpack.canvas.seriesStyle for all ids in this file